dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolReferenceSet.java

```diff
@@ -52,6 +53,14 @@ static Builder builder() {
     abstract ImmutableSet.Builder<ClassSymbolReference> classReferencesBuilder();
     abstract ImmutableSet.Builder<MethodSymbolReference> methodReferencesBuilder();
     abstract ImmutableSet.Builder<FieldSymbolReference> fieldReferencesBuilder();
-    abstract SymbolsInFile build();
+
+    abstract SymbolReferenceSet build();
+
+    Builder merge(SymbolReferenceSet other) {
```

Comment:
Maybe "add" instead of "merge"? Up to you.

Reply:
`Merge` makes more sense to me as the parameter is the same class.

Reply:
You could also do "addAll", just like Set has "addAll". Totally up to you.

Reply:
That sounds better. Now it's `addAll`.